ConnectorNavisworks/ConnectorNavisworks/Bindings/ConnectorNavisworksBindings.Filters.cs

```diff
@@ -30,7 +30,7 @@ public override List<ISelectionFilter> GetSelectionFilters()
 
     var selectSetsRootItem = _doc.SelectionSets.RootItem;
 
-    var savedSelectionSets = selectSetsRootItem.Children.Select(GetSets).ToList();
+    var savedSelectionSets = selectSetsRootItem?.Children.Select(GetSets).ToList() ?? new List<TreeNode>();
```

Comment:
Will Children always be non-null in these two linq statements also?

Reply:
API documentation says it will be an empty list